docs/troubleshooting-oauth2.md

```diff
@@ -23,15 +23,15 @@ limitations under the License.
 
 ## Overview {#overview}
 
-This guide covers the most common errors encountered using OAuth 2.0 authentication protocol in OAuth 2.0 authentication backend and management plugin and how to diagnose them.
+This guide covers the most common errors encountered using [OAuth 2.0](./oauth2/) and management plugin and how to diagnose them.
 
 ## Troubleshooting OAuth 2 in the management UI {#management-ui}
```

Comment:
Either keep upper case or lower capitalization in headings: Table of Contents is all caps on first instance of a letter in a word, this one is lower case, please check what is mainly done in other OSS RMQ docs and work with that one and keep consistent throughout